app/src/main/res/xml/preferences.xml

```diff
@@ -28,10 +28,10 @@
             android:icon="@drawable/ic_network" />
 
     <Preference
-            android:key="prefScreenGpodder"
+            android:key="prefScreenSynchronization"
             android:title="@string/synchronization_pref"
             android:summary="@string/synchronization_sum"
-            android:icon="@drawable/ic_star" />
+            android:icon="@drawable/ic_notification_sync" />
```

Comment:
The star icon was used on purpose because otherwise users might confuse feed refresh and external synchronization. Please revert back to the star icon for now.